vips/type.go

```diff
@@ -43,9 +43,6 @@ type Color struct {
 
 const DefaultFont = "sans 10"
 
-// ColorBlack is shorthand for black RGB
-var ColorBlack = Color{0, 0, 0}
```

Comment:
Looks like a useful constant for users of this package

Reply:
fine